README.md

```diff
@@ -18,427 +18,15 @@ A modern, lightweight learning management system.
 
 ## Getting started
 
-### Getting started on OS X
+### Getting started on MacOS, Linux or Docker.
 
-#### 1. Install Homebrew and Homebrew Cask
+#### 1. Install script
```

Comment:
No need to number install script... we had this for the relative sections under getting started.

Install script should be under its own section, i.e.:

```md
### Install Script

Blah... 

### Manual Install

#### 1. Step one title

Do this

#### 2. Step two title

Do that
```

etc.